modules/heroku/main.tf

```diff
@@ -8,9 +8,9 @@ resource "heroku_app" "heroku" {
   organization {
     name = data.heroku_team.heroku.name
   }
-  buildpacks = [
+  buildpacks = concat([
```

Comment:
https://devcenter.heroku.com/articles/using-multiple-buildpacks-for-an-app#adding-a-buildpack
> The buildpack for the primary language of your app should always be the last buildpack in the list. This ensures that defaults for that primary language are applied instead of those for another language, and allows Heroku to correctly detect the primary language of your app.